src/Templates/src/templates/maui-mobile/Resources/Styles.xaml

```diff
@@ -61,10 +61,6 @@
             <VisualStateGroupList>
                 <VisualStateGroup x:Name="CommonStates">
                     <VisualState x:Name="Normal">
-                        <VisualState.Setters>
```

Comment:
Could we include some kind of TODO: comment to avoid forgetting to add this again after fixing AppThemeBinding?

Reply:
Leaving this blank is the correct behavior. This way when you travel back to normal in unsets back to the values set on the control itself.  If you have setters inside `Normal` then it causes code like this

`<Button TextColor=Green />`

To no longer work, because the VSM will just use the VSM setters over the property set on the control itself.